src/lib/password-strength/constant/passwordGraph.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
i suggest not to format the json files to keep them small

Reply:
yeah that a good idea, thank you for catching this